IPython/core/completer.py

```diff
@@ -544,45 +545,45 @@ def back_latex_name_matches(text):
 
 class IPCompleter(Completer):
     """Extension of the completer class with IPython-specific features"""
-    
+
     @observe('greedy')
     def _greedy_changed(self, change):
         """update the splitter and readline delims when greedy is changed"""
         if change['new']:
             self.splitter.delims = GREEDY_DELIMS
         else:
             self.splitter.delims = DELIMS
-    
+
     merge_completions = Bool(True,
         help="""Whether to merge completion results into a single list
-        
+
         If False, only the completion results from the first non-empty
         completer will be returned.
         """
     ).tag(config=True)
     omit__names = Enum((0,1,2), default_value=2,
         help="""Instruct the completer to omit private method names
-        
```

Comment:
Could we ask you to redo this without stripping all the whitespace from these files? Sorry, but it messes up git blame. If you're using atom, you can stop it doing this in Preferences > Packages > Whitespace.